src/lightning_app/cli/lightning_cli.py

```diff
@@ -50,6 +51,89 @@ def _main():
     pass
 
 
+@_main.group("create")
```

Comment:
Might be worth to start organizing cli into sub-files if more is being added.

Reply:
I agree - for now I've split up list, delete and create into separate files.